src/users_db.rs

```diff
@@ -111,7 +111,7 @@ impl UserBuilder {
         self
     }
 
-    pub fn is_admin(&mut self, admin: bool) -> &mut UserBuilder {
+    pub fn set_admin(&mut self, admin: bool) -> &mut UserBuilder {
```

Comment:
This is due to: [wrong_self_convention](https://github.com/Manishearth/rust-clippy/wiki#wrong_self_convention)